### ISSUE TITLE
Feature Request - Separate Zips for Each World

### ISSUE THREAD

**Issue by SERVERCONSOLE** [Exolius/SimpleBackup#23]
It would be great to be able to have a separate archive for each world. I have a 5G backup and it is a bit unwieldy as a single file.

**Comment by godsyn** [comment]
I too would like to request this feature. eg:
$backupdir/$worldname/$time.zip, so it is compatible with worldedit.

**Comment by chamunks** [comment]
Oh man @godsyn turns out that it is actually compatible with WorldEdit as it is.  This project still functions but is pretty abandoned.  

`WorldEdit/config.yml example`

    snapshots:
        directory: backups/

**Comment by Exolius** [comment]
Yeah this is pretty much on hold update wise unless it becomes hilariously broken, mianly because of my loss of intrest in Minecraft.
